fmriprep/interfaces/images.py

```diff
@@ -124,7 +124,27 @@ def _run_interface(self, runtime):
         return runtime
 
 
-CONFORMSERIES_TEMPLATE = """\t\t<h3 class="elem-title">Anatomical Conformation</h3>
+class TemplateDimensionsInputSpec(BaseInterfaceInputSpec):
+    t1w_list = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input T1w images')
+
+    max_scale = traits.Float(3.0, usedefault=True,
+                             desc='Maximum scaling factor in images to accept')
```

Comment:
Nice! Do you know if it is possible to enforce this pattern of attr spacing via flake?